templates/activity.html

```diff
@@ -1,41 +1,75 @@
-<!doctype html>
-<html>
-
-<head>
-    <title>Community Activity</title>
-    <style>
-    canvas {
-        -moz-user-select: none;
-        -webkit-user-select: none;
-        -ms-user-select: none;
-    }
-    </style>
-    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-</head>
-
-<body>
-    <h1>Community Activity</h1>
-    <br>
-    <br>
-    <div style="width:75%;">
-        <canvas id="canvas"></canvas>
+<!DOCTYPE html>
+  <html>
+    <head>
+      <!--Import Google Icon Font-->
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+      <!--Import materialize.css-->
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+      <link rel="stylesheet" href="../static/css/style.css">
+      <!--Let browser know website is optimized for mobile-->
+      <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    </head>
+
+    <body>
+
+    <nav>
+    <div class="nav-wrapper">
+      <a href="#" class="brand-logo">&nbsp;<img src="https://api.coala.io/en/latest/_static/images/coala_logo.svg"></a>
```

Comment:
no. The objective was to solve https://github.com/coala/community/issues/42 , fixing https://github.com/coala/community/issues/1 .

Reply:
So should i remove the logo? 😟

Reply:
I've referred this issue in the commit message. Will that work?